tests/Sorry.Analyzers.Tests/RR1337IfStatementInTestMethodTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿namespace Sorry.Analyzers.Tests
+{
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using RoslynTestKit;
+    using Xunit;
+
+    public class RR1337IfStatementInTestMethodTests : AnalyzerTestFixture
+    {
+        protected override string LanguageName => LanguageNames.CSharp;
+
+        [Fact]
+        public void XunitFactAttributeTest()
+        {
+            const string text = @"
+namespace Tests;
+
+using Xunit;
+
+public class Foo
+{
+    [Fact]
+    public void MyTestMethod()
+    {
+        [|if (true)|]
+        {
+        }
+    }
+}
+";
+
+            this.HasDiagnostic(
+                markupCode: text,
+                diagnosticId: DiagnosticIds.Sorry1337);
+        }
+
+        [Fact]
+        public void XunitTheoryAttributeTest()
```

Comment:
I used a different naming style for test methods. I don’t insist, but what if using the same naming notation is a good idea?

Reply:
I'm so sorry. I didn't notice your naming style here. Fixed.